crates/miden-testing/src/tx_context/context.rs

```diff
@@ -67,11 +67,13 @@ impl TransactionContext {
         code: &str,
         assembler: Assembler,
     ) -> Result<Process, ExecutionError> {
+        // TODO: Remove this unwrap
         let (stack_inputs, advice_inputs) = TransactionKernel::prepare_inputs(
             &self.tx_inputs,
             &self.tx_args,
             Some(self.advice_inputs.clone()),
-        );
+        )
+        .unwrap();
```

Comment:
Not sure we can do something better than panic here without changing the error return type, which would be quite a refactoring I imagine. I'd change this to an `expect` here so that if it fails we get a slightly better error at least.